subsys/net/lib/coap/coap_client.c

```diff
@@ -27,6 +27,7 @@ static atomic_t coap_client_recv_active;
 static int send_request(int sock, const void *buf, size_t len, int flags,
 			const struct sockaddr *dest_addr, socklen_t addrlen)
 {
+	LOG_HEXDUMP_DBG(buf, len, "Send CoAP Request:");
```

Comment:
I think this is a bit too much debug information?
Is this intended to be left here or was it just a left over from development? We can debug CoAP messages properly in Wireshark if needed.

Reply:
Yes, it may be too much at times. It is inconvenient now to change this as my upstream PR merged already.

Ideally we had another log level -- in some systems I have seen "insane" in addition to "debug".

In this case, I probably should have gated it with a macro.